From eb45bdfe66af7844a779bc6fcf923cd951336309 Mon Sep 17 00:00:00 2001
From: Dennis Bonke <admin@dennisbonke.com>
Date: Fri, 6 Oct 2023 10:39:45 +0200
Subject: [PATCH 19/25] libcamera: pipeline: simple: Enable simplepipeline for
 intel-ipu6 DNU

Do Not Upstream, first the ipu6 CSI receiver code needs to land in
the kernel.

Signed-off-by: Dennis Bonke <admin@dennisbonke.com>
Signed-off-by: Hans de Goede <hdegoede@redhat.com>
---
 src/libcamera/pipeline/simple/simple.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libcamera/pipeline/simple/simple.cpp b/src/libcamera/pipeline/simple/simple.cpp
index c76510c2..130843cd 100644
--- a/src/libcamera/pipeline/simple/simple.cpp
+++ b/src/libcamera/pipeline/simple/simple.cpp
@@ -197,6 +197,7 @@ static const SimplePipelineInfo supportedDevices[] = {
 	{ "mxc-isi", {} },
 	{ "qcom-camss", {} },
 	{ "sun6i-csi", {} },
+	{ "intel-ipu6", {} },
 };
 
 } /* namespace */
-- 
2.43.0

